deployAWS/invoke_lambda.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
I'd prefer to have just one consolidated `scripts` folder rather than both `deployAWS` and `deployLOCAL`.